CHANGELOG.md

```diff
@@ -16,7 +16,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Features:
 - The GDK now uses SpatialOS `13.8.1`.
-- Automatic local deployment starting. Local deployments are now started automatically for you when pressing the 'Play' button. Local deployment start time has been reduced to around 5.5s~. If your schema has changed during a deployment, the next time you press play the local deployment will be automatically restarted. There is no longer a `spatial` cmd window for a local deployment, the Unreal output window will still contain logs. Runtime logs can be found at `spatial\logs\localdeployment\%timestamp%\runtime.log`. A new option `Show spatial service button` in the SpatialOS Settings menu allows you to turn the 'spatial service' on and off via the SpatialGDK Toolbar for debugging purposes. Each time you start the editor in any project the 'spatial service' will be restarted, this is to ensure the service is always running in the correct SpatialOS project. You can disable this auto start feature via the new SpatialOS setting "Auto-start local deployment".
+- Local deployment startup time has been significantly reduced.
+- Local deployments now start automatically when you select `Play`. This means you no longer need to select `Start` in the GDK toolbar before you select `Play` in the Unreal toolbar.
+- If your schema has changed dureing a local deployment, the next time you select `Play` the deployment will automatically restart.
+- Local deployments no longer run in a seperate Command Prompt, they now run in the Unreal Editor's Output Log.
```

Comment:
"they now run in the Unreal Editor's Output Log" -> "logs are now found in the Unreal Editor's Output Log"

Reply:
fixed